### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4573,24 +4573,93 @@ condtition->condition
...
+conecctivities->connectivities
+conecctivity->connectivity
+conecctor->connector
+conecctors->connectors
+coneccts->connects
+conecept->concept
+conecepts->concepts
+conecjture->conjecture
+conecjtures->conjectures
+conecnt->content, connect,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: i vote for `connect` to be the first result.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4573,24 +4573,93 @@ condtition->condition
...
+conecntrate->concentrate
+conecntrated->concentrated
+conecntrates->concentrates
+conecnts->contents, connects,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: same here

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -4573,24 +4573,93 @@ condtition->condition
...
+conenctor->connector
+conenctors->connectors
+conencts->connects
+conenience->convenience
+conenient->convenient
+coneninece->convenience
+coneninet->convenient
 conent->content
 conents->contents
+coner->corner
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `coroner`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -4573,24 +4573,93 @@ condtition->condition
...
+conergence->convergence
+conern->concern
+conerning->concerning
+coners->corners
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `coroners`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -4573,24 +4573,93 @@ condtition->condition
...
+coneting->connecting
+conetion->connection
+conetions->connections
+conetivities->connectivities
+conetivity->connectivity
+conetnt->content
+conetor->connector
+conetors->connectors
+conets->connects
 conetxt->context
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please add `connects`

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -4573,24 +4573,93 @@ condtition->condition
...
+conetxts->contexts
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please add `connects`

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -4573,24 +4573,93 @@ condtition->condition
...
 conexant->connexant
 conext->context, connect, connects,
+conexts->contexts
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: add `connects`

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -6893,7 +6962,14 @@ dignostics->diagnostics
 dilema->dilemma
 dillema->dilemma
 dimemsions->dimensions
+dimenion->dimension
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: add `dominion`

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -12914,6 +13048,7 @@ memroy->memory
 menally->mentally
 menas->means
 menetion->mention
+menioned->mentioned
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: add  
`menetioned->mentioned`  
`menetions->mentions`  
`menion->mention`  
`menions->mentions`

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -16677,6 +16820,7 @@ readd->readd, re-add, read,
 readded->readded, read,
 readeable->readable
 readed->read, readd, readded,
+READEME->README
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: should be `reademe->README`

**Comment by pabs3** [comment]
Did the changes requested.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by luzpaz** [comment]
@pabs3 Thanks. Heads up, when you make lots of changes like that, it's best to have it as a separate commit so we can check more conveniently to see what was done. In other words, when you squash all in to one we don't know what you actually removed or added.

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]:
```
menion->mention, minion,
menions->mentions, minions,
```
Wasn't added

**Comment by luzpaz** [comment]
PR ready for merge

**Pull request merged**